src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx

```diff
@@ -195,6 +204,10 @@ export default function InvestOption({ claim, optionIndex, openModal, closeModal
       error = ErrorMsgs.InsufficientBalance(token?.symbol)
     } else if (!isNative && !isApproved) {
       error = ErrorMsgs.NotApproved(token?.symbol)
+    } else if (noBalance) {
+      error = ErrorMsgs.InsufficientBalance(token?.symbol)
+    } else if (!parsedAmount && !isTouched) {
+      error = ''
```

Comment:
all the others are validation checks, and this one is kind if an exception. I would add one comment to clarify, why u do `error = ''` it could look kike you just forgot to add the text there.

Reply:
Done

Reply:
Where's the comment?

Reply:
Removed it with a revert, now its back